tools/cross/install_rootfs.sh

```diff
@@ -1,9 +1,13 @@
 #!/usr/bin/env bash
+
+source /etc/lsb-release
+
 usage()
 {
     echo "Usage: $0 [BuildArch] [LinuxCodeName] [--setproxy=IP] [--skipunmount]"
     echo "BuildArch can be: arm(default), aarch64"
-    echo "LinuxCodeName - optional, Code name for Linux, can be: bionic, focal(default), jammy"
+    echo "LinuxCodeName - optional, Code name for Linux, can be: bionic, focal, jammy"
```

Comment:
Now there is no default OS version in this script. OS version follows the host system. :)